File: LeetCode/LeetCode 75/151. Reverse Words in a String.java
class Solution {
    public String reverseWords(String s) {
       String[] words = s.trim().split(" ");
       String ans = "";
       for(int i = words.length - 1; i>=0; i--) {
           if (!words[i].isEmpty()) {
               ans += (words[i] + " ");
           }
       }

       return ans.substring(0, ans.length() - 1);
    }
}

File: Leetcode/Leetcode 75/151. Reverse Words in a String.js
/**
 * @param {string} s
 * @return {string}
 */
var reverseWords = function(s) {
    let words = s.split(' ');
    let ans = [];
    for (let i = words.length - 1; i>=0; i--) {
        if (words[i]) {
            ans.push(words[i]);
        }
    }
    return ans.join(' ');
};
